src/components/Avatar/Avatar.tsx

```diff
@@ -34,83 +35,124 @@ export interface Props {
   accessibilityLabel?: string;
 }
 
+export interface State {
```

Comment:
Do we always export our `State` interfaces?

Reply:
yes we do